### PR TITLE
chore(deps): update dev dependencies

### PR DESCRIPTION
**Pull request by bnjm**
Attempt #3 at bumping these deps 🤠 Initially I thought it had introduced a regression with conflicting peer deps:

```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: @vitejs/plugin-react@2.2.0
npm WARN Found: vite@4.4.7
npm WARN node_modules/vite
npm WARN   dev vite@"^4.4.7" from the root project
npm WARN   6 more (@joshwooding/vite-plugin-react-docgen-typescript, ...)
...
```

... but that conflict already exists on the main branch. So it makes sense to separate from #66

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,29 +77,31 @@
...
     "@vitejs/plugin-vue": "^4.2.3",
-    "@vitest/ui": "^0.33.0",
+    "@vitest/ui": "^0.34.1",
     "@vue/eslint-config-prettier": "7.1.0",
     "@vue/eslint-config-typescript": "^11.0.3",
     "autoprefixer": "^10.4.14",
     "chromatic": "^6.20.0",
-    "eslint": "^8.45.0",
+    "eslint": "^8.46.0",
+    "eslint-config-prettier": "^9.0.0",
```
Comment (manuatarchilogic) [inline package.json]: a new dependency?

Comment (bnjm) [reply package.json]: I installed it to resolve a peer conflict and forgot to remove it again, thanks [a992c1b](https://github.com/archilogic-com/honeycomb/pull/67/commits/a992c1bc3040977bc41b1d08badf3aca3b737899)

```sh
@archilogic/honeycomb@2.4.2 
├─┬ @archilogic/eslint-config@2.0.0
│ └── eslint-config-prettier@9.0.0
└─┬ @vue/eslint-config-prettier@7.1.0
  └── eslint-config-prettier@8.10.0
```

File: package.json [hunk 2]
```diff
@@ -77,29 +77,30 @@
...
     "@vue/eslint-config-typescript": "^11.0.3",
     "autoprefixer": "^10.4.14",
     "chromatic": "^6.20.0",
-    "eslint": "^8.45.0",
+    "eslint": "^8.46.0",
     "eslint-plugin-prettier": "^5.0.0",
-    "eslint-plugin-vue": "^9.15.1",
+    "eslint-plugin-vue": "^9.16.1",
     "husky": "^8.0.3",
+    "jsdom": "^22.1.0",
```
Comment (luk-zap) [inline package.json:94]: do we need `jsdom`?

Comment (bnjm) [reply package.json:94]: yeah, it's used by Vitest (see `vite.config.ts#test`). I think it was previously auto-installed, but after the latest update I was prompted to explicitly install it

**Review by luk-zap — APPROVED** [review]

**Review by romiekos — APPROVED** [review]

**Pull request merged**